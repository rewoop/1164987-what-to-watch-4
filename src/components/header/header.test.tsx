import * as React from "react";
import * as renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import history from "../../history";
import Header from "./header";

it(`Should Header render correctly`, () => {
  const tree = renderer
    .create(
        <Router history={history}>
          <Header
            isSignIn={`AUTH`}
          />
        </Router>).toJSON();

  expect(tree).toMatchSnapshot();
});
