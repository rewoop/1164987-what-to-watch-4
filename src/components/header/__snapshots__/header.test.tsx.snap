// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Header render correctly 1`] = `
Array [
  <h1
    className="visually-hidden"
  >
    WTW
  </h1>,
  <header
    className="page-header movie-card__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="user-block"
    >
      <a
        href="/mylist"
        onClick={[Function]}
      >
        <div
          className="user-block__avatar"
        >
          <img
            alt="User avatar"
            height="63"
            src="img/avatar.jpg"
            width="63"
          />
        </div>
      </a>
    </div>
  </header>,
]
`;
